pennylane/measurements/counts.py

```diff
@@ -56,6 +56,11 @@ def counts(op=None, wires=None, all_outcomes=False) -> "CountsMP":
     :math:`p(\lambda_i) = |\langle \xi_i | \psi \rangle|^2`, where :math:`| \xi_i \rangle`
     is the corresponding basis state from the observable's eigenbasis.
 
+    .. note::
+
+        Differentiation of QNodes that return ``counts`` is currently not supported. Please refer to
+        :func:`~.pennylane.sample` if differentiability is required.
+
```

Comment:
Pretty sure you can't differentiate with respect to counts either.

Reply:
Oh I see the comment below. Single shot expectation value.

Reply:
Should we then go back to how it was before [this update](https://github.com/PennyLaneAI/pennylane/pull/4283#discussion_r1239029103)?